monai/data/png_writer.py

```diff
@@ -48,7 +48,7 @@ def write_png(data,
         assert isinstance(output_shape, (list, tuple)) and len(output_shape) == 2, \
             'output_shape must be a list of 2 values (H, W).'
 
-        if len(data.shape) > 2:
+        if 2 < len(data.shape) < 5:
```

Comment:
Why < 5 here for 2D image? This is the length of dimensions, not channel number.
Thanks.

Reply:
Sorry. You are right. ==3 makes sense.

Reply:
Changed accordingly.